cmake/FileUtil.cmake

```
@@ -41,13 +49,20 @@ function(ly_include_cmake_file_list file)
                 list(APPEND UNITY_AUTO_EXCLUSIONS ${f})
             endif()
         endif()
-
     endforeach()
+
     list(APPEND FILES ${file}) # Add the _files.cmake to the list so it shows in the IDE
 
     if(file_path)
-        list(TRANSFORM SKIP_UNITY_BUILD_INCLUSION_FILES PREPEND ${file_path}/)
-    endif()
+        foreach(f ${SKIP_UNITY_BUILD_INCLUSION_FILES})
+            cmake_path(IS_RELATIVE f is_relative)
+            if(is_relative)
+                string(PREPEND f ${file_path}/)
+            endif()
```

repeat: [cmake_path(APPEND)](https://cmake.org/cmake/help/latest/command/cmake_path.html#append) should be used here.

```suggestion
            cmake_path(APPEND file_path ${f} OUTPUT_VARIABLE f)
```